src/views/FarmAuction/components/AuctionLeaderboard/AuctionLeaderboardTable.tsx

```diff
@@ -0,0 +1,220 @@
+import styled from 'styled-components'
+import BigNumber from 'bignumber.js'
+import {
+  Text,
+  Flex,
+  Box,
+  OpenNewIcon,
+  BunnyPlaceholderIcon,
+  Skeleton,
+  Button,
+  useTooltip,
+  useMatchBreakpoints,
+  useModal,
+} from '@pancakeswap/uikit'
+import { getBscScanLink } from 'utils'
+import { getBalanceNumber } from 'utils/formatBalance'
+import EllipsisMenu, { OptionProps } from 'components/EllipsisMenu/EllipsisMenu'
+import { useTranslation } from 'contexts/Localization'
+import { usePriceCakeBusd } from 'state/farms/hooks'
+import { Auction, Bidder } from 'config/constants/types'
+import WhitelistedBiddersModal from '../WhitelistedBiddersModal'
+
+const LeaderboardContainer = styled.div`
+  display: grid;
+  grid-template-columns: 1fr 5fr 3fr 1fr;
+  ${({ theme }) => theme.mediaQueries.md} {
+    grid-template-columns: 3fr 5fr 5fr 1fr;
+  }
+`
+
+const GridCell = styled(Flex)<{ isTopPosition }>`
+  height: 65px;
+  align-items: center;
+  border-bottom: 1px solid ${({ theme }) => theme.colors.cardBorder};
+
+  ${({ theme, isTopPosition }) => isTopPosition && `background-color: ${theme.colors.warning}2D;`}
+`
+
+const BidderEllipsisMenu: React.FC<{ bidder: Bidder }> = ({ bidder }) => {
+  const { t } = useTranslation()
+  const handleSortOptionChange = (option: OptionProps) => {
+    window.open(option.value, '_blank').focus()
+  }
+
+  const { projectSite, lpAddress, account } = bidder
+
+  const options = []
+  if (projectSite) {
+    options.push({
+      label: t('Project Site'),
+      value: projectSite,
+      icon: <OpenNewIcon />,
+    })
+  }
+  if (lpAddress) {
+    options.push({
+      label: t('LP Info'),
+      value: `http://pancakeswap.info/pool/${lpAddress}`,
+      icon: <OpenNewIcon />,
+    })
+  }
+  options.push({
+    label: t('Bidder Address'),
+    value: getBscScanLink(account, 'address'),
+    icon: <OpenNewIcon />,
+  })
+  return <EllipsisMenu options={options} onChange={handleSortOptionChange} />
+}
+
+interface LeaderboardRowProps {
+  topLeaderboard: number
+  bidder: Bidder
+  cakePriceBusd: BigNumber
+  isMobile: boolean
+}
+
+const LeaderboardRow: React.FC<LeaderboardRowProps> = ({ topLeaderboard, bidder, cakePriceBusd, isMobile }) => {
+  const { t } = useTranslation()
+  const { tooltip, targetRef, tooltipVisible } = useTooltip(
+    <Text>
+      {t('Top %num% bidders at the end of the auction will successfully create a community farm.', {
+        num: topLeaderboard,
+      })}
+    </Text>,
+    { placement: 'bottom' },
+  )
+  const { isTopPosition, position, samePositionAsAbove, farmName, tokenName, amount } = bidder
+  return (
+    <>
+      <GridCell isTopPosition={isTopPosition} pl={['12px', '24px']}>
+        <Flex ref={targetRef}>
+          <Text bold={isTopPosition} color={samePositionAsAbove ? 'textDisabled' : 'text'} textTransform="uppercase">
+            #{position}
+          </Text>
+          {tooltipVisible && isTopPosition && tooltip}
```

Comment:
Is this in the spec? Seems strange to have the same tooltip for every row.

Reply:
It is the same yes, but it is only shown winning positions. Will ask Bugs about it, originally there were trophy icons next to top position rows, but we decided to scratch that so now its not even clear that there is tooltip....

Reply:
Talked to bugs, ditched these tooltips - 3ea2e216ef95f85ac1431ef2037e4dbc9ae7c00d